<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>replaceForm</class>
 <widget class="QWidget" name="replaceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>173</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="findLabel">
            <property name="text">
             <string>&amp;Find</string>
            </property>
            <property name="buddy">
             <cstring>findText</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="replaceLabel">
            <property name="text">
             <string>Replace &amp;with</string>
            </property>
            <property name="buddy">
             <cstring>replaceText</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="findText"/>
          </item>
          <item>
           <widget class="QLineEdit" name="replaceText"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="replaceButton">
            <property name="text">
             <string>&amp;Replace</string>
            </property>
            <property name="autoRepeat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="moreButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>&amp;More</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="closeButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>x</string>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="optionsGroupBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="caseCheckBox">
           <property name="text">
            <string>Case sensitive</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="backwardsCheckBox">
           <property name="text">
            <string>Backwards</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="selectedCheckBox">
           <property name="text">
            <string>Selected text only</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="wholeWordsCheckBox">
           <property name="text">
            <string>Whole words</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="promptCheckBox">
           <property name="text">
            <string>Prompt on repalce</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findText</tabstop>
  <tabstop>replaceText</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>moreButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>caseCheckBox</tabstop>
  <tabstop>wholeWordsCheckBox</tabstop>
  <tabstop>backwardsCheckBox</tabstop>
  <tabstop>promptCheckBox</tabstop>
  <tabstop>selectedCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>optionsGroupBox</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moreButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>optionsGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
